CMakeLists.txt

```diff
@@ -199,13 +199,13 @@ set(ENERGYPLUS_VERSION_MINOR 1)
 set(ENERGYPLUS_VERSION_PATCH 0)
 set(ENERGYPLUS_VERSION "${ENERGYPLUS_VERSION_MAJOR}.${ENERGYPLUS_VERSION_MINOR}.${ENERGYPLUS_VERSION_PATCH}")
 # Build SHA is not required to have a value, but if it does OpenStudio will require this build.
-set(ENERGYPLUS_BUILD_SHA "68a4a7c774")
+set(ENERGYPLUS_BUILD_SHA "1c11a3d85f")
 
 # ENERGYPLUS_RELEASE_NAME is used to locate the E+ download
 # from the github releases
-set(ENERGYPLUS_RELEASE_NAME "v25.1.0")
+set(ENERGYPLUS_RELEASE_NAME "v25.1.0-WithDSOASpaceListFixes")
 
-set(ENERGYPLUS_REPO "NREL")
+set(ENERGYPLUS_REPO "jmarrec")
```

Comment:
Use a custom E+ build from my fork

https://github.com/jmarrec/EnergyPlus/releases/tag/v25.1.0-WithDSOASpaceListFixes

This is 25.1.0 with two extra PRs:

* https://github.com/NREL/EnergyPlus/pull/11053
* https://github.com/NREL/EnergyPlus/pull/11051